utils/cliutils/commandsflags.go

```diff
@@ -1223,6 +1230,10 @@ var flagsMap = map[string]cli.Flag{
 		Name:  InsecureTls,
 		Usage: "[Default: false] Set to true to skip TLS certificates verification, while encrypting the Artifactory password during the config process.` `",
 	},
+	projectPath: cli.StringFlag{
+		Name:  projectPath,
+		Usage: "[Optional] A full path for a user project. ` `",
```

Comment:
Usage: "[Optional] A full path for a user project. ` `",
-->
Usage: "[Default: ./] Full path to the code project. ` `",